src/Geosuggest.jsx

```diff
@@ -414,58 +438,66 @@ class Geosuggest extends React.Component {
    */
   render() {
     const attributes = filterInputAttributes(this.props),
-      classes = classnames(
-        'geosuggest',
-        this.props.className,
-        {'geosuggest--loading': this.state.isLoading}
-      ),
+      classes = classnames('geosuggest', this.props.className, {
```

Comment:
I prefer to have `const` for every new variable. It's more readable for me.

just my preference :)

Reply:
This would be a refactoring, not just a tooling upgrade.